src/it/scala/io/iohk/ethereum/ledger/BlockImporterItSpec.scala

```diff
@@ -150,7 +152,7 @@ class BlockImporterItSpec
     blockImporter ! BlockFetcher.PickedBlocks(NonEmptyList.fromListUnsafe(newBranch))
 
     //because the blocks are not valid, we shouldn't reorganise, but at least stay with a current chain, and the best block of the current chain is oldBlock4
-    eventually { blockchain.getBestBlock().get shouldEqual oldBlock4 }
+    eventually {Thread.sleep(1000); blockchain.getBestBlock().get shouldEqual oldBlock4 }
```

Comment:
wouldn't it be cleaner to adjust the `implicit def patienceConfig: PatienceConfig = defaultPatienceConfig`?

Reply:
@dzajkowski I have tried both mixing our `LongPatience` trait into the test, and defining patience config directly before the `eventually` block. From the debug, I see that the my values are used, but for some reason method is not re-run.

Reply:
interesting, let me give it a look

Reply:
`eventually` does not mix well with `AsyncFlatSpecLike`. either a `Future` and `AsyncFlatSpecLike` or `AnyFlatSpecLike` with `eventually`

Reply:
@dzajkowski  Done.